azure-pipelines.yml

```diff
@@ -0,0 +1,32 @@
+trigger:
+  branches:
+    include: ['*']
+  tags:
+    include: ['*']
+
+jobs:
+  - job: Test
+  pool:
+    vmImage: 'Ubuntu-18.04'
```

Comment:
That's OK to test on Ubuntu-18.04 but I would want to have Ubuntu-16.04 too
Which are all the other available VMs? (including mac and windows?)

Reply:
Ok, i will do 16.04.

Reply:
I would like to keep 18 and add 16. And also possibly any other OS that they may have available

Reply:
thanks. I see you have added 14 and 16.